projects/igniteui-angular/src/lib/grids/filtering/advanced-filtering/advanced-filtering-dialog.component.html

```diff
@@ -183,29 +183,21 @@ <h6 class="igx-filter-empty__title">
                            [(ngModel)]="searchValue"/>
                 </igx-input-group>
 
-                <igx-date-picker *ngIf="selectedColumn && selectedColumn.dataType === 'date'"
-                                 mode="dropdown"
-                                 [(ngModel)]="searchValue"
-                                 [locale]="grid.locale"
-                                 [outlet]="grid.outlet">
-                    <ng-template igxDatePickerTemplate let-openDialog="openDialog" let-value="value">
-                        <igx-input-group #dropDownTarget type="box" [displayDensity]="'compact'">
-                            <input #searchValueInput
-                                    igxInput
-                                    tabindex="0"
-                                    (keydown)="invokeClick($event)"
-                                    (click)="openDialog(dropDownTarget.element.nativeElement)"
-                                    [placeholder]="grid.resourceStrings.igx_grid_filter_row_date_placeholder"
-                                    autocomplete="off"
-                                    [value]="
-                                        selectedColumn.formatter
-                                        ? (value | columnFormatter:selectedColumn.formatter)
```

Comment:
Do we support a `formatter` function for the value?

Reply:
Yes, we support the `formatter` function, it is now added to the templates. :)